packages/bee-q/src/components/avatar/scss/bq-avatar.variables.scss

```
@@ -3,8 +3,42 @@
 /* -------------------------------------------------------------------------- */
 
 :host {
+  /**
+  * @prop --bq-avatar--background: Avatar background color
+
+  * @prop --bq-avatar--border-color: Avatar border color
+  * @prop --bq-avatar--border-style: Avatar border style
+  * @prop --bq-avatar--border-width: Avatar border width
+  * @prop --bq-avatar--border-radius: Avatar border radius
+
+  * @prop --bq-avatar--size-xsmall: Avatar xsmall size
+  * @prop --bq-avatar--size-small: Avatar small size
+  * @prop --bq-avatar--size-medium: Avatar medium size
+  * @prop --bq-avatar--size-large: Avatar large size
+
+  * @prop --bq-avatar--badge-top: Badge top position
+  * @prop --bq-avatar--badge-right: Badge right position
+  * @prop --bq-avatar--badge-left: Badge left position
+  */
+
+  --bq-avatar-background: theme('colors.ui.secondary-light');
+
+  --bq-avatar--border-color: theme('colors.stroke.tiertary');
+  --bq-avatar--border-style: solid;
+  --bq-avatar--border-width: theme('borderSpacing.xs3');
```

Please use spacing tokens only for internal paddings/margins in the component:

```suggestion
  --bq-avatar--border-width: 2px;
```